src/cljs/nr/gameboard.cljs

```diff
@@ -1375,6 +1374,20 @@
                                     (audio-sfx "run-successful")
                                     (audio-sfx "run-unsuccessful")
                                     (audio-sfx "virus-purge")))]
+        (r/create-class
+            {:display-name "audio-component"
+             :component-did-update
+             (fn []
+                 (update-audio {:sfx (:sfx @game-state) :sfx-current-id (:sfx-current-id @game-state)
+                                :gameid (:gameid @game-state)} soundbank))
+             :reagent-render
+             (fn [{:keys [sfx] :as cursor}]
+              (let [_ @sfx]) ;; hack: make this component rebuild when sfx changes. Could probably move audio to its own component.
```

Comment:
Could update comment now that audio is its own component.

Reply:
Is the code still necessary? I'm not entirely sure what rebuilding the component in this case does.

Reply:
Updated comment.